.github/ISSUE_TEMPLATE/config.yaml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.iota.org/
```

Probably better to link to the builders discord
```suggestion
    url: https://discord.com/invite/iota-builders
```